app/shared/app-data/src/commonMain/kotlin/data/source/media/fetch/MediaFetcher.kt

```diff
@@ -358,3 +378,26 @@ class MediaSourceMediaFetcher(
         private const val ENABLE_WATCHDOG = false
     }
 }
+
+@JvmInline
+value class CompletedConditions(
```

Comment:
```suggestion
class CompletedConditions(
```

这个东西看起来会用在被 box 的地方, 那其实不需要 `value`